data/core.yaml

```diff
@@ -1589,6 +1589,11 @@ en:
         reference: "All roads, paths, and ferry routes should connect to form a single routing network."
       highway:
         message: "{highway} is disconnected from other roads and paths"
+    duplicate_address:
+      title: "Duplicate Addresses"
+      message: "{entity1} and {entity2} contain similar addresses"
+      tip: "Find potentially duplicate addresses."
+      reference: "Addresses should not share the same housenumber, unless they differ in other attributes like floor or unit"
```

Comment:
I know 'housenumber' refers to an address tag part (`addr:housenumber`) but is there a better plain english term we could use? Or maybe we just say 'Addresses should not share the same number (addr:housenumber), unless they differ in other attributes like floor or unit'

I think that second one might be a bit clearer to somebody who's unfamiliar with addr tag contents.

Reply:
Also, maybe add a guidance for fixing like "Please review the addr: tags on these two entities carefully and consider if it makes sense to merge all the address information into one and delete the other."

Reply:
Good question!  I always struggle over the best way to write these messages for a wide audience.  OSMers will recognize "housenumber" "floor" and "unit" as tags.. 

I _think_ lay users would know what a "housenumber" is, or that this string could be translated in a way that makes sense in different languages.  

Also I wanted to avoid putting the literal tags in the message in case we want to adjust the validation later to check more esoteric tags like "conscriptionnumber" or whatever tags people use around the world.